### PR TITLE
Only include hand written code in jacoco report

### PR DESCRIPTION
**Pull request by sacheu**

### CODE REVIEW

**Review by minnieliu — APPROVED** [review]

File: sdk/communication/pom.xml [hunk 1]
```diff
@@ -57,7 +64,7 @@
                 </goals>
                 <configuration>
                   <outputDirectory>${project.reporting.outputDirectory}/test-coverage</outputDirectory>
-                </configuration>
+            </configuration>
```
Comment (JianpingChen) [inline sdk/communication/pom.xml:67]: NIT: restore tab

**Review by JianpingChen — APPROVED** [review]
There is one formatting change that may not be intentional.

File: sdk/communication/pom.xml (hunk 1 above)
Comment (sacheu) [reply sdk/communication/pom.xml]: Thanks. Fixed.

**Pull request merged**